examples/lead-generation/src/graph/graph.ts

```diff
@@ -0,0 +1,94 @@
+import { CSV_GENERATOR_NODE_NAME, csvGeneratorNode } from "@/graph/nodes/csv-generator-node";
+import { ERROR_HANDLER_NODE_NAME, errorHandlerNode } from "@/graph/nodes/error-handler-node";
+import { OUTREACH_MESSAGE_NODE_NAME, outreachMessageNode } from "@/graph/nodes/outreach-message-node";
+import { ENRICH_THERAPISTS_NODE_NAME, enrichTherapistNode } from "@/graph/nodes/therapist-enrichment-node";
+import { FETCH_THERAPISTS_NODE_NAME, fetchTherapistsNode } from "@/graph/nodes/therapist-fetcher-node";
+import { URL_VALIDATOR_NODE_NAME, urlValidatorNode } from "@/graph/nodes/url-validator-node";
+import { ConfigurableAnnotation, type LeadGenerationGraphConfig, StateAnnotation, type Therapist } from "@/graph/state";
+import { AirtopClient } from "@airtop/sdk";
+import { END, START, StateGraph } from "@langchain/langgraph";
+import { ChatOpenAI } from "@langchain/openai";
+export type LeadGenerationGraphResult = {
+  therapists: Therapist[];
+  csvContent: string;
+  csvPath: string;
+  error: string;
+};
+
+/**
+ * Lead Generation First Graph
+ *
+ * This graph is used to generate pull the initial data of the therapists from the provided websites.
+ * It will validate the URLs, fetch the therapists, and handle any errors.
+ * The output of this graph will later be passed to the second graph.
+ *
+ */
+export const leadGenerationGraphPart1 = async (
```

Comment:
Nit: I think naming these graphs something more descriptive like `leadGenerationSourcingGraph` and  `leadGenerationEnrichmentGraph` instead of Part1 and Part2 would help with clarity and readability.